src/pkg/caendr/caendr/models/sql/phenotype_metadata.py

```diff
@@ -11,7 +11,7 @@ class PhenotypeMetadata(DictSerializable, db.Model):
 
   trait_name_caendr = db.Column(db.String(), unique=True, primary_key=True)
   trait_name_user = db.Column(db.String(), nullable=True)
-  species = db.Column(db.String())
+  species_name = db.Column(db.String())
```

Comment:
Note -- when deploying this code to production, we need clear path to do the migration (renaming of column name), and ensuring it won't break. Test in lower envs with existing data.